docs/source/training/socialisation/run_intermediate_inasafe.rst

```diff
@@ -0,0 +1,396 @@
+.. _run_intermediate_inasafe:
+
+.. image:: /static/training/socialisation/inasafe_logo.*
+
+Run Intermediate |project_name|
+===============================
+
+Introduction
+------------
+
+In `Run Basic InaSAFE <http://docs.inasafe.org/en/training/socialisation/run_basic_inasafe.html/>`__ we learned how to run flood analysis on
+population and buildings, understand flood impact default settings and
+impact summary report, and how to change analysis threshold. We also
+learned how to save our work and generate a pdf of the analysis results.
+In this exercise we will learn more techniques such as how to run
+|project_name| with aggregation data to produce more specific results. And we
+will learn how to download data using the OpenStreetMap downloader and
+how to run |project_name| analysis on the downloaded building points and roads.
+Last but not least, we will learn how to define the analysis area.
+
+Learning Objective
+------------------
+
+To improve participant’s understanding of the |project_name| function and
+features. By the end of this exercise, participants will be able to:
+
+- Define keywords using the |project_name| keyword wizard
+
+- Run |project_name| with aggregation data
+
+- Use the OpenStreetMap Downloader to download data for |project_name| and
+
+- Set the |project_name| analysis area using the |project_name| analysis area tool.
+
+Data for this exercise
+----------------------
+
+DKI Jakarta.zip from `|project_name| Training
+Data <http://data.inasafe.org/TrainingDataPackages/>`__ and we will use
+the following data:
+
+1. Jakarta Flood 18113 WGS84
+
+2. Jakarta District Boundary WGS84
+
+3. Jakarta Subdistrict Boundary WGS84
+
+4. Jakarta Population WGS84
+
+Exercise
+--------
+
+Run |project_name| for Population Data
+......................................
+
+A. Add and symbolize vector data
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Before we start, click :guilabel:`Open` and select :file:`DKI Jakarta Intermediate.qgs`.
+If there is notification to save the current project, you can choose
+either to save the project or not. It will look like this:
+
+.. image:: /static/training/socialisation/intermediate_inasafe_01.*
+   :align: center
+   :width: 300 pt
+
+Next, add :file:`Jakarta Flood 18113 WGS84` data from :file:`Run Intermediate InaSAFE > DKI Jakarta > 01 Hazard Data`. This is the flood hazard data
+for Jakarta from an actual flood event. During Jakarta floods in January
+2013, provincial disaster manager collects information about flood,
+including the location of flooded areas by sub-village boundary
+(`see more about this data at… <http://docs.inasafe.org/en/training/socialisation/datasets.html>`__).
+
+Let us symbolize this layer so that it shows only affected areas (if you
+forget how to symbolize, you can go to `Introduction to QGIS <http://docs.inasafe.org/en/training/socialisation/introduction_to_qgis.html/>`__).
```

Comment:
as above - use :ref: and `section reference` for internal references http://sphinx-doc.org/markup/inline.html